components/Forms/ComponentFactory.tsx

```diff
@@ -91,7 +92,12 @@ export const ComponentFactory: React.VFC<FactoryProps> = observer(
             className="col-span-2"
             noValidate
           >
-            <input type="hidden" name="_language" value={'EN'} />
+            <input
+              type="hidden"
+              name="_language"
+              id="_language"
+              value={router.locale == 'en' ? Language.EN : Language.FR}
```

Comment:
In the backend, Locale is like `en-CA` and `fr-CA`, while Language is `EN` or `FR`.  How come you are checking for a locale of `en`? Shouldn't it be `en-CA`? If not, then why not use Language instead of Locale, so you don't need these sorts of conversions?

Reply:
(this comment applies elsewhere as well)